### PR TITLE
PHIOS-6358 [Motion Free] Q3 Implement Platform virtual vehicle API support for Onboarding

### PR DESCRIPTION
**Pull request by alex-kepar**
- added supporting values "MojioType" and "VehicleType" according to https://1mojio.atlassian.net/wiki/spaces/AR/pages/1200259114/Motion-Free+APIs#POST-{domain-prefix}-api.moj.io/mojios/{mojioType}

https://1mojio.atlassian.net/browse/PHIOS-6358

### CODE REVIEW

File: MojioSDK/Models/Vehicles/Vehicle.swift [hunk 1]
```diff
@@ -16,6 +16,18 @@
...
 import MojioCore
 
+public enum VehicleType: String, Codable {
+    case physical = "Physical"
+    case ghost = "Ghost"
+    case bluetooth = "Bluetooth"
+    case unknown = "Unknown"
+
+    public init(from decoder: Decoder) throws {
+        let label = try decoder.singleValueContainer().decode(String.self)
```
Comment (andriimakukha) [inline MojioSDK/Models/Vehicles/Vehicle.swift]: Maybe `value` or `stringValue` is better?

File: MojioSDK/Models/Vehicles/Mojio.swift [hunk 2]
```diff
@@ -16,6 +16,17 @@
 import Foundation
 import MojioCore
 
+public enum MojioType: String, Codable {
+    case obd2 = "Obd2"
+    case virtual = "Virtual"
+    case unknown = "Unknown"
+
+    public init(from decoder: Decoder) throws {
+        let label = try decoder.singleValueContainer().decode(String.self)
```
Comment (andriimakukha) [inline MojioSDK/Models/Vehicles/Mojio.swift]: Maybe `value` or `stringValue` is better?

**Review by andriimakukha — APPROVED** [review]

**Review by vladyslavpetruk-epam — APPROVED** [review]

File: MojioSDK/Models/Vehicles/Vehicle.swift (hunk 1 above)
Comment (alex-kepar) [reply MojioSDK/Models/Vehicles/Vehicle.swift]: @andriimakukha  Thanks. It renamed to rawValue.

File: MojioSDK/Models/Vehicles/Mojio.swift (hunk 2 above)
Comment (alex-kepar) [reply MojioSDK/Models/Vehicles/Mojio.swift]: @andriimakukha  Thanks. It renamed to rawValue.

**Pull request merged**